rules/windows/file_event/sysmon_detect_powerup_dllhijacking.yml

```diff
@@ -0,0 +1,24 @@
+title: Powerup Write Hijack DLL detection
+id: 602a1f13-c640-4d73-b053-be9a2fa58b96
+status: experimental
+description: Powerup tool's Write Hijack DLL exploits DLL hijacking for privilege escalation. In it's default mode, it builds a self deleting .bat file which executes malicious command. The detection rule relies on creation of the malicious bat file (debug.bat by default). Reference: https://powersploit.readthedocs.io/en/latest/Privesc/Write-HijackDll/
+author: pbssubhash
+date: 2021/08/21
+modified: 2021/08/21
+tags:
+    - attack.persistence
+    - attack.privilege_escalation
+    - attack.defense_evasion
+    - attack.t1574.001
+logsource:
+    category: file_event
+    product: windows
+detection:
+    selection:
+        Image|endswith: '\powershell.exe'
+        File.path|endswith: '.bat'
```

Comment:
Use the original field name thanks

Reply:
The field according to the event log would be `TargetFileName`. However, when I use sigmac to convert the rule, it doesn't convert properly. Is it okay?
![image](https://user-images.githubusercontent.com/6551267/130324167-8823730f-ee2d-4d21-97eb-07bfdb0483cc.png)

Reply:
Modified as advised. kindly check, thanks.